examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<blockquote>activity <code>wait_before_acquire_async(time_s)</code></blockquote><p>This is not the activity but the activity_func</p><p> </p><blockquote>utilities for long time</blockquote><p>"for a longer time" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>